### DISCUSSION TITLE
Feature Request: Rename All References (Refactoring within Editor)

### DISCUSSION THREAD

**Discussion by yulrun** [godotengine/godot-proposals#8463] [Scripting]
The main thing I find missing from Godot's scripting is the ease of refactoring that an IDE like VSCode or Rider offer, such as renaming a function in a class, and all references using that function get updated. Currently its a whole process of trying to either A) Track down all references manually, or B) run the game and wait for errors to find the references.

**Comment by Zireael07** [comment]
Already open on the tracker as https://github.com/godotengine/godot-proposals/issues/899
